http/src/main/resources/org/broadinstitute/dsde/workbench/leonardo/liquibase/changesets/20220118_update_runtime_controlled_resource.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="leonardo" xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="leonardo" author="jdcanas" id="update_runtime_controlled_resource">
+        <addColumn tableName="RUNTIME_CONTROLLED_RESOURCE">
+            <column name="azureName" type="VARCHAR(254)">
```

Comment:
I think its important since 
   1) its required for any querying of a controlled resource in azure itself (the ComputeManager `get` calls
   2) to provide readable logging down the line that can be easily mapped to azure entities as opposed to needing to look up IDs from Leo in WSM first

It's not super useful as of this PR (especially since for VM and PDs we have the name stored in their respective tables already), but I think it's a good idea to store. I could see an argument for removing it for now though, not too attached.

Reply:
Is there an example of this where we don't already store the info? cuz say for VM, the vm name should already exist in `Runtime` table; for disk name, it should already be in `PersistentDisk` table, and for these 2, it'll be duplicate info if we add this column here...so I'm inclined to not have it unless we have an example that we do need this field

Reply:
For Network and IP we don't store this info anywhere else. That being said, we don't actually need this field for anything yet. I'lll remove it.